CHANGES.md

```diff
@@ -5,5 +5,51 @@ First stable release.
 
 # grmtools 0.1.1 (2018-12-18)
 
-* Fix bug where %epp strings with quote marks in caused a code-generation
+* Fix bug where `%epp` strings with quote marks in caused a code-generation
   failure in compile-time mode.
+
+
+# grmtools 0.2.0 (2019-01-14)
+
+## Breaking changes
+
+* Previously we forced users to specify the `YaccKind` of a grammar and then the
+  `ActionKind` of actions. This is unnecessarily fiddly, so remove `ActionKind`
```

Comment:
> so **we** remove**d** `ActionKind`  entirely and instead flesh**ed** out `YaccKind`

Reply:
I've tried an alternative in the commit below, removing "we", because it seems a bit odd (in retrospect) to personalise things.

Reply:
Nevermind, I missed the bullets points without which these sentences sounded weird. I see now what style you were going for. Maybe change the "we" in the first sentence to "grmtools" instead, i.e. "Previously grmtools forced users..."